### PR TITLE
Small cleanup before making the repo public

### PR DESCRIPTION
**Pull request by nathanaelbosch**
So far I updated the README just to be a little bit more complete.

What's left to do: Should all the scripts in `./experiments` actually be there, or should some of them be deleted? 

EDIT: I just deleted `./experiments/old/` since I assume they are not used anywhere relevant.

### CODE REVIEW

**Comment by pnkraemer** [comment]
From memory: should be fine as is. Do you want me to take a closer look?


(Also, maybe you should update your black version. But I might be wrong.)

**Comment by nathanaelbosch** [comment]
Not it should be fine, we clarified this offline :) I thought about moving the files for the inset plots to a different location though, since they are not really part of the "experiment" per se.

**Comment by pnkraemer** [comment]
oops, must've forgotten. Sorry :smile:

**Comment by pnkraemer** [comment]
> I thought about moving the files for the inset plots to a different location though, 

I would be fine with that btw :)

File: experiments/4_vdp_stiffness_comparison.py [hunk 1]
```diff
@@ -134,7 +136,10 @@ def main():
     if not RESULT_DIR.is_dir():
         RESULT_DIR.mkdir(parents=True)
     df = pd.DataFrame(result_dict)
-    df.to_csv(RESULT_DIR / "results.csv", sep=";", index=False)
+    result_file = RESULT_DIR / "results.csv"
+    df.to_csv(result_file, sep=";", index=False)
+
+    plotting.plot_4_vdp_stiffness_comparison(result_file)
```
Comment (nathanaelbosch) [inline experiments/4_vdp_stiffness_comparison.py:142]: I added this since all the other scripts also do that

**Review by nathanaelbosch — COMMENTED** [review]
If you have a better proposal for the scripts that create the system trajectories, as in better names or a better location, let me know :)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,34 @@
-# high-dimensional-ode-solver-experiments
+# "Probabilistic ODE Solvers in Millions of Dimensions" - Experiments
 
-Short: "hose".
\ No newline at end of file
+This repo contains the experiment code for the paper "Probabilistic ODE Solvers in Millions of Dimensions", which is accepted at ICML 2022, currently avaiable on [arXiv](https://arxiv.org/abs/2110.11812).
```
Comment (pnkraemer) [inline README.md]:
```suggestion
This repo contains the experiment code for the paper "Probabilistic ODE Solvers in Millions of Dimensions", which is accepted at ICML 2022, and currently available on [arXiv](https://arxiv.org/abs/2110.11812).
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,34 @@
...
+
+## Run Experiments
+Before running anything, install the package with
```
Comment (pnkraemer) [inline README.md]:
```suggestion
Before running the scripts, install the package with
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,34 @@
...
+```
+pip install -e .
+```
+
+`./experiments/` contains the scripts to run the experiments and create the corresponding figure.
```
Comment (pnkraemer) [inline README.md]:
```suggestion
`./experiments/` contains the scripts to run the experiments and create the corresponding figures.
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,34 @@
...
+
+
+## Reference
+The paper is currently on [arXiv](https://arxiv.org/abs/2110.11812):
+```
+@misc{https://doi.org/10.48550/arxiv.2110.11812,
+  doi = {10.48550/ARXIV.2110.11812},
+  url = {https://arxiv.org/abs/2110.11812},
+  author = {Krämer, Nicholas and Bosch, Nathanael and Schmidt, Jonathan and Hennig, Philipp},
+  keywords = {Machine Learning (stat.ML), Machine Learning (cs.LG), Numerical Analysis (math.NA), FOS: Computer and information sciences, FOS: Computer and information sciences, FOS: Mathematics, FOS: Mathematics},
```
Comment (pnkraemer) [inline README.md]: is this necessary? I always remove this kind of info, because I think it clutters the whole entry quite a lot

File: README.md [hunk 6]
```diff
@@ -1,3 +1,34 @@
...
+  title = {Probabilistic ODE Solutions in Millions of Dimensions},
+  publisher = {arXiv},
+  year = {2021},
```
Comment (pnkraemer) [inline README.md:30]: Just checking: we don't have a 2022 version yet, right?

Comment (nathanaelbosch) [reply README.md:30]: nope

File: README.md [hunk 7]
```diff
@@ -1,3 +1,34 @@
...
+
+
+## Reference
+The paper is currently on [arXiv](https://arxiv.org/abs/2110.11812):
+```
+@misc{https://doi.org/10.48550/arxiv.2110.11812,
+  doi = {10.48550/ARXIV.2110.11812},
+  url = {https://arxiv.org/abs/2110.11812},
+  author = {Krämer, Nicholas and Bosch, Nathanael and Schmidt, Jonathan and Hennig, Philipp},
+  keywords = {Machine Learning (stat.ML), Machine Learning (cs.LG), Numerical Analysis (math.NA), FOS: Computer and information sciences, FOS: Computer and information sciences, FOS: Mathematics, FOS: Mathematics},
```
Comment (nathanaelbosch) [inline README.md]:
```suggestion
```

File: README.md (hunk 5 above)
Comment (nathanaelbosch) [reply README.md]: removed it

**Review by pnkraemer — APPROVED** [review]
Thank you!

**Review by schmidtjonathan — APPROVED** [review]
🙏

**Pull request merged**